setup.py

```diff
@@ -6,30 +6,31 @@
 from __future__ import absolute_import, print_function
 
 import os
-import re
 from distutils.sysconfig import get_config_var, get_python_inc
 
 import versioneer
 from Cython.Distutils.build_ext import new_build_ext as build_ext
 from setuptools import setup
 from setuptools.extension import Extension
 
-include_dirs = [os.path.dirname(get_python_inc())]
-library_dirs = [get_config_var("LIBDIR")]
-libraries = ["ucp", "uct", "ucm", "ucs"]
-extra_compile_args = ["-std=c99", "-Werror"]
+# Patch versioneer version for wheel builds.
+if "RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE" in os.environ:
+    orig_get_versions = versioneer.get_versions
+
+    version_override = os.environ["RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE"]
 
+    def get_versions():
+        data = orig_get_versions()
+        data["version"] = version_override
+        return data
 
-def get_ucp_version():
-    with open(include_dirs[0] + "/ucp/api/ucp_version.h") as f:
-        ftext = f.read()
-        major = re.findall("^#define.*UCP_API_MAJOR.*", ftext, re.MULTILINE)
-        minor = re.findall("^#define.*UCP_API_MINOR.*", ftext, re.MULTILINE)
```

Comment:
Just so we're on the same page, I've already removed this in this PR because it was unused. I assume you're just explaining why it can go, not requesting any particular changes?

Reply:
yep